metadata.json

```diff
@@ -66,7 +66,7 @@
       "version_requirement": ">= 5.5.10 < 7.0.0"
     }
   ],
-  "pdk-version": "1.14.1",
+  "pdk-version": "1.15.0.pre (heads/master-0-gf5fdf1a)",
```

Comment:
Shouldn't be using non-released versions of PDK on production modules.

Reply:
all the changes outside sync file were done by pdk update